indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunnerFactory.java

```diff
@@ -34,6 +34,7 @@
  */
 public class ForkingTaskRunnerFactory implements TaskRunnerFactory<ForkingTaskRunner>
 {
+  public static final String TYPE_NAME = "local";
```

Comment:
so I think whether it's "local" or whatever properly belongs to the Overlord/MM guice modules and not to the runner factory itself.